src/interfaces/tests/CMakeLists.txt

```diff
@@ -15,10 +20,15 @@ add_library(
    STATIC
    ${CMAKE_SOURCE_DIR}/src/utils/memory.c
```

Comment:
memory utils should not be used. They will be removed once the `refactor` branch is finished with development.